model2vec/inference/model.py

```diff
@@ -192,6 +214,27 @@ def evaluate(
 
         return report
 
+    def predict_logits(self, token_ids: list[int]) -> np.ndarray:
+        """Predict the logits for the specified token IDs."""
+        # Extract embeddings for the specified token IDs.
+        token_embeddings = self.embeddings[token_ids]
+        mlp = self.head[-1]
```

Comment:
just a fair warning that this doesn't work if the head is ever not a pipeline with an mlp as final estimator. Which we do support, in principle.